source/reporting/index.html.md.erb

```diff
@@ -399,19 +399,29 @@ You can use custom metadata to add more information to new payments when creatin
 
 Paying users cannot see metadata while they're making a payment.
 
-Include a `metadata` object in the body of your request to [create a new payment](/making_payments/#take-a-payment), for example:
+Include a `metadata` object in the body of your request to [create a new payment](/api_reference/create_a_payment_reference). 
 
-```javascript
-"metadata": {
-   "ledger_code": "AB100",
-   "an_internal_reference_number": 200,
-...
+This example request creates a payment for a £100 fine and adds a ledger code and an internal reference number as metadata:
+
+`POST /v1/payments`
+
+```json
+{
+  "amount": 10000,
+  "reference" : "WA5678",
+  "description": "Pay a fine for a late tax filing.",
+  "return_url": "https://your.service.gov.uk/completed",
+  "language": "cy",
+  "metadata": {
+    "ledger_code": "AB100",
+    "an_internal_reference_number": 200
+  }
 }
 ```
 
-The `metadata` object must contain between 1 and 10 parameters.
+The `metadata` object must contain between 1 and 10 parameters as key-value pairs.
 
-Each parameter key must be a unique case-insensitive string between 1 and 30 characters long. If 2 or more keys are identical, the API will remove all but one of the identical keys from the `metadata` object.
+Each parameter key must be a unique, case-insensitive string between 1 and 30 characters long. If 2 or more keys are identical, the API will remove all but one of the identical keys from the `metadata` object.
```

Comment:
is it "between 1 and 30" or "one and 30"?

SG: "Use ‘one’ unless you’re talking about a step, a point in a list or another situation where using the numeral makes more sense".

Technically it's none of the specific items, but I would argue "makes more sense". Just thought I'd flag anyway, as to me it's *borderline*

Reply:
Good flag. This might be complete nonsense but I think `1` is acceptable because it's a technical content? If I changed it in this line, I think I'd need to change it in the next line:

> Each parameter key must be a unique, case-insensitive string between 1 and 30 characters long

and I think when we're getting into talking about string length in API calls, keeping it as a number makes more sense.